### PR TITLE
Home page changes

### PR DESCRIPTION
**Pull request by indytechcook**
Just a few small changes to the home page and removing some logging for the demo.

@vietnogi @tladendo

### CODE REVIEW

File: src/client/app/getting-started/views/getting-started.jade [hunk 1]
```diff
@@ -1,26 +1,9 @@
...
-- builds.push({ href: '#', icon: 'project-development', label: 'Mobile app development'})
-- builds.push({ href: '#', icon: 'project-custom', label: 'Custom solution'})
-
 section.jumbotron
   p Our Talented 70,000 strong community is ready to work on your project today
 
 header
   h1 What can we build for you?
 
-main.choose-build
```
Comment (vietnogi) [inline src/client/app/getting-started/views/getting-started.jade]: need to remove css for this stuff

Comment (indytechcook) [inline src/client/app/getting-started/views/getting-started.jade]: Thanks, It's been removed.

**Comment by tladendo** [comment]
lgtm

**Comment by vietnogi** [comment]
lgtm

**Pull request merged**